src/HistoryQuery/HistoryQueryRepository.class.js

```diff
@@ -0,0 +1,162 @@
+const sqlite = require('sqlite')
+const sqlite3 = require('sqlite3')
+
+class HistoryQueryRepository {
+  static TABLE = 'history_queries'
+
+  constructor(databasePath) {
+    this.databasePath = databasePath
+    this.database = null
+  }
+
+  /**
+   * Initialize the HistoryQuery database
+   * @return {Promise<void>} - The result promise
+   */
+  async initialize() {
+    this.database = await sqlite.open({ filename: this.databasePath, driver: sqlite3.cached.Database })
+    const query = `CREATE TABLE IF NOT EXISTS ${HistoryQueryRepository.TABLE} (
+                   id TEXT PRIMARY KEY,
+                   order INTEGER,
+                   name TEXT,
+                   status TEXT,
+                   enabled INTEGER,
+                   paused INTEGER,
+                   south_id TEXT,
+                   north_id TEXT,
+                   start_time TEXT,
+                   end_time TEXT,
+                   file_pattern TEXT,
+                   compress INTEGER,
+                   settings TEXT
+                 );`
+    const stmt = await this.database.prepare(query)
+    await stmt.run()
+  }
+
+  /**
+   * Create a new HistoryQuery.
+   * @param {object} historyQuery - The HistoryQuery info
+   * @return {Promise<object>} - The HistoryQuery
+   */
+  async create(historyQuery) {
+    const query = `INSERT INTO ${HistoryQueryRepository.TABLE} (id, order, name, status, enabled, paused, southId, northId, startTime, endTime,
+                                                                filePattern, compress, settings)
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+    const stmt = await this.database.prepare(query)
+    await stmt.run(
+      historyQuery.uuid,
```

Comment:
We should use id instead of uuid to be consistent with other entities of the project.